opencti-platform/opencti-front/src/private/components/common/stix_core_objects/StixCoreObjectMappableContent.tsx

```diff
@@ -59,7 +59,7 @@ const StixCoreObjectMappableContent: FunctionComponent<StixCoreObjectMappableCon
 }) => {
   const { t_i18n } = useFormatter();
   let { description, contentField } = containerData;
-  const [selectedTab, setSelectedTab] = useState(editionMode ? 'write' : 'preview');
```

Comment:
Not sure to understand this change. It does not seem relevant to me to ignore if we are in edition. Why not keeping this line but setting `editionMode` to false in the parent ?

Reply:
We want to always have "preview" as default, even in edition. But may be I'm missing something, can you give more insight of your thought ?

Reply:
Maybe I misunderstood what is asked by the issue. I found odd to set mode view only when entering in edition mode

Reply:
That is what is expected by the product.